code/_onclick/click.dm

```diff
@@ -346,23 +346,19 @@
 	icon_state = "click_catcher"
 	plane = CLICKCATCHER_PLANE
 	mouse_opacity = 2
-	screen_loc = "CENTER-7,CENTER-7"
+	screen_loc = "CENTER"
 
-/obj/screen/click_catcher/proc/MakeGreed()
-	. = list()
-	for(var/i = 0, i<15, i++)
-		for(var/j = 0, j<15, j++)
-			var/obj/screen/click_catcher/CC = new()
-			CC.screen_loc = "NORTH-[i],EAST-[j]"
-			. += CC
+/obj/screen/click_catcher/New()
+	..()
+	transform = matrix(200, 0, 0, 0, 200, 0)

```

Comment:
Make it 200 times as big as a single tile. It was bothering me during debugging when I was trying to find the plane master objects in client.screen and it was like 50 click catcher objects.

I actually forgot I had done this. If you want it in another PR, that's fair.